scripts/scil_compare_connectivity.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Performs a network-based statistical comparison for populations X and Y using
+a t-statistic threshold of alpha. All matrices must have the same shape and be
+ordered in the same way.
+
```

Comment:
Can you provide an example how you call this script please like you sometimes do?